pkg/Makefile

```diff
@@ -43,7 +43,62 @@ osc-cli-x86_64.zip: osc-cli
 osc-cli:
 	$(SUDO) docker run --rm -v ${PWD}/..:/src outscale/wine-pyinstaller:0.0.1 --distpath ./pkg --clean --name osc-cli osc_sdk/sdk.py
 
+fcu.json:
+	curl -s https://git.unice.fr/com/foundation_email_edit/-/raw/8702ba3efcb35ab7c4d9240d620582cc52b79eeb/node_modulesold/aws-sdk-apis/apis/ec2-2014-06-15.api.json > fcu.json
```

Comment:
I suppose this specific version is hard to be found online as it is a bit old.
Please secure this file by importing it in this commit.

Reply:
+ check original project license